Sources/Sentry/SentryReplayRecording.m

```diff
@@ -0,0 +1,64 @@
+#import "SentryReplayRecording.h"
+#import "SentryDateUtil.h"
+
+@implementation SentryReplayRecording
+
+- (instancetype)initWithSegmentId:(NSInteger)segmentId
+                             size:(NSInteger)size
+                            start:(NSDate *)start
+                         duration:(NSTimeInterval)duration
+                       frameCount:(NSInteger)frameCount
+                        frameRate:(NSInteger)frameRate
+                           height:(NSInteger)height
+                            width:(NSInteger)width
+{
+    if (self = [super init]) {
+        self.segmentId = segmentId;
+        self.size = size;
+        self.start = start;
+        self.duration = duration;
+        self.frameCount = frameCount;
+        self.frameRate = frameRate;
+        self.height = height;
+        self.width = width;
+    }
+    return self;
+}
+
+- (nonnull NSArray<NSDictionary<NSString *, id> *> *)serialize
+{
+
+    long timestamp = [SentryDateUtil javascriptDate:self.start];
+
+    NSDictionary *metaInfo = @{
+        @"type" : @4,
+        @"timestamp" : @(timestamp),
+        @"data" : @ { @"href" : @"", @"height" : @(self.height), @"width" : @(self.width) }
+    };
+
+    NSDictionary *recordingInfo = @{
+        @"type" : @5,
+        @"timestamp" : @(timestamp),
+        @"data" : @ {
+            @"tag" : @"video",
+            @"payload" : @ {
+                @"segmentId" : @(self.segmentId),
+                @"size" : @(self.size),
+                @"duration" : @(self.duration),
+                @"encoding" : @"h264",
+                @"container" : @"mp4",
```

Comment:
how about moving this to some ReplayConstant we can refer also at the place where we do the recording. In the event we need to change anything it's easy to find out what needs to change

Reply:
We can move this, but being very pragmatic, this will very unlikely ever change when it comes to video recording on iOS, and probably not used anywhere else.